com.microsoft.mrtk.graphicstools.unity/Samples~/MaterialGallery/Scripts/EmissiveRotation.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections;
```

Comment:
I bet you could replace this script with an animation by animating the texture offset's `W` component:
![Anim](https://user-images.githubusercontent.com/13305729/177884522-7d825b3e-29d2-4dcd-b50f-22796b86c1cd.gif)

Might be a bit safer too since as time gets large, we will run into floating point precision issues.